e2e/vertical-full-page-map.spec.ts

```diff
@@ -49,20 +62,32 @@ test.describe('full page map test suite', () => {
   });
 
   test('default initial search works and is enabled by default', async ({ page }) => {
-    await page.getByPlaceholder('Search for locations').press('Enter');
-    const result = page.locator('#js-answersVerticalResults div').nth(0);
-    await expect(result).toBeAttached();
+    const response = await page.waitForResponse(resp =>
+      resp.url().includes('https:\/\/prod-cdn\.us\.yextapis\.com\/v2\/accounts\/me\/search\/vertical\/query')
+      && resp.url().includes('queryTrigger'));
+    expect(response.status()).toBe(200);
   });
 
+  test('empty search works', async ({ page }) => {
+    await page.getByPlaceholder('Search for locations').press('Enter');
+    const response = await page.waitForResponse(resp =>
+      resp.url().includes('https:\/\/prod-cdn\.us\.yextapis\.com\/v2\/accounts\/me\/search\/vertical\/query')
+      && resp.url().includes('input=&'));
+    expect(response.status()).toBe(200);
+  })
+
   test('pagination works', async ({ page }) => {
+    const firstPage = page.locator('#js-answersVerticalResultsCount');
```

Comment:
looks like the `firstPage` and `secondPage` locators are the same. can we just use one of them and call it `resultsCount` or something?